### ISSUE TITLE
✅ 🔗 🚀 Feature Request: Table of contents widget minimize box option (Duplicate of #9933)

### ISSUE THREAD

**Issue by kdtb** [elementor/elementor#10968]
**What problem is your feature request going to solve? Please describe.**
I am simply looking for a option to minimize the TOC widget as default when you visit a site. I have a hard time understanding which webmaster would be interested in a TOC that is by default opened/extended. For me, I always want the box to be closed.

**Describe the solution you'd like**
I am simply looking for a option to minimize the TOC widget as default when you visit a site. I am using TOC on every page, but I don't want the box to be opened unless my visitors click on the box in order to open/extend it.
I want the box to look like this, when i visit my site:
![image](https://user-images.githubusercontent.com/62666996/77575229-89f65b00-6ed3-11ea-883e-fa30364eed50.png)

**Describe alternatives you've considered**
Well, there are a lot of plugins. I think this is a really easy feature to add to the widget.

**Comment by WDUK1** [comment]
I would like to be able to do this as well.  As long as Google can still read the contents for SEO :-)

**Comment by iStuff0** [comment]
I would like this as well. Maybe there is some HTML or shortcode it can be done with, like we can do with the Accordion?

**Comment by bkjproductions** [comment]
In the latest Elementor it has an option to start out miminized, but unfortunately it briefly shows full-height and then shrinks down. This doesn't look so hot when the page loads.

![problemn-with-thing-showing-up](https://user-images.githubusercontent.com/4551281/174161630-61296b82-f4e3-4a82-8e3f-197dc9ae08f3.gif)

**Comment by aatu86** [comment]
Yes, this is still a problem, as referenced here in a now-closed thread: https://github.com/elementor/elementor/issues/9933

You can make the TOC closed by default on any device including desktop, but it starts open and then collapses, which is visually disturbing and bad for CLS (cumulative layout shift), a Google ranking criteria

**Comment by nicholaszein** [comment]
Hello there!

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.7.0! 🥳 

You can now set the Table of Contents to be closed on Desktop as well.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
